src/test/java/com/google/devtools/build/lib/skyframe/BUILD

```diff
@@ -433,3 +434,23 @@ java_test(
         "//third_party:truth",
     ],
 )
+
+java_test(
+    name = "TrackSourceDirectoriesIntegrationTest",
+    srcs = [":TrackSourceDirectoriesIntegrationTest.java"],
```

Comment:
Can I drop the `:` as part of the import? We don't use those for files.

Reply:
Sure